net/FlatBuffers/FlatBuffers.Core.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <Compile Remove="Properties\**" />
```

Comment:
There are two "better" ways of doing what you are doing here.

1. (preferred for now) Is to add `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` to your `PropertyGroup` above. This will disable the automatic generation of the AssemblyInfo.cs file that the Microsoft.NET.Sdk does automatically. Then you can remove all these `ItemGroup`s here. The advantage to this approach is that the assemblies produced by both `FlatBuffers.Core.csproj` and `FlatBuffers.csproj` will have the same assembly level attributes.  As it stands now, because you aren't setting things like `<Copyright>` in this project, you aren't getting the same assembly attribute in the compiled assembly as the other project does. See [here](https://github.com/google/flatbuffers/blob/master/net/FlatBuffers/Properties/AssemblyInfo.cs#L29).
2. Long term, a better way would be to delete the `Properties/AssemblyInfo.cs` file all together and add any specific properties (like Copyright, Version, and AssemblyFileVersion) to this .csproj. Then the SDK can just generate the AssemblyInfo.cs file with the properties from this .csproj.

Check out this [blog](https://compiledexperience.com/blog/posts/assemblyinfo-generation) and the [public docs](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#assemblyinfo-properties) for more info.

Reply:
Thank you Eric, will check it out